src/component/recipeAddForm/recipeFormModalTypes.ts

```diff
@@ -0,0 +1,30 @@
+export interface RecipeFormState {
+  title: string;
+  source_url: string;
```

Comment:
[non-blocking] [naming] I don't think we should use snake case. Unless it's the request body that we send to backend, most of the time we use camelCase